yaml/csr.yaml

```diff
@@ -2576,127 +2659,32 @@
             msb: 31
             lsb: 28
             values:
```

Comment:
This format does nothing for the generated tests. They are a custom extension we made up and I don't know what more to do about it.